features/customers.feature

```diff
@@ -364,4 +364,78 @@ Scenario: Update a Customer with Missing Password
     And I should see "Required field" in the "Password" error string
     When I set the "Password" to "test"
     And I press the "Update" button
-    Then The "Password" error string should be gone 
\ No newline at end of file
+    Then The "Password" error string should be gone 
+
+Scenario: Delete a Customer that Exists
+    When I visit the "Home Page"
+    And I set the "First Name" to "Bob"
```

Comment:
We should ideally search and retrieve a customer already created in the background rather than creating a new one.
Hopefully it doesn't break the integration tests in pipeline. We can revert if it breaks them.